sanic_openapi/autodoc.py

```diff
@@ -0,0 +1,93 @@
+import inspect
+import warnings
+
+import yaml
+
+
+class OpenAPIDocstringParser(object):
```

Comment:
Inheriting from object isn't necessary isn't it?

Reply:
True, fixed